src/cascadia/WpfTerminalTestNetCore/WpfTerminalTestNetCore.csproj

```diff
@@ -1,13 +1,23 @@
-<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-    <RuntimeFrameworkVersion>3.1.18</RuntimeFrameworkVersion>
+    <TargetFramework>net6.0-windows10.0.22621.0</TargetFramework>
     <UseWPF>true</UseWPF>
     <Platforms>AnyCPU;x64;x86</Platforms>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <CsWinRTIncludes>Microsoft.Terminal.Core</CsWinRTIncludes>

```

Comment:
There may be an easier way to handle this -- I am going to push a couple fixes to your branch and the merge it! Thanks again for making this update :)